### PR TITLE
feature/discussion polls

### PR DESCRIPTION
**Pull request by jamestjw**
- Make it possible to add polls in discussions
- Implement voting in polls and displaying of poll results

### CODE REVIEW

**Comment by jamestjw** [comment]
I recommend running it to see if it works :)

File: install.php [hunk 1]
```diff
@@ -12,9 +12,8 @@
 // Load database tables
 try {
     $connection = new PDO("mysql:host=$host", $username, $password, $options);
-    $database_name = $environment."_cga";
     $sql = file_get_contents("data/init.sql");
-    $sql = sprintf($sql, $database_name);
+    $sql = sprintf($sql, $dbname);
     $connection->exec($sql);
```
Comment (jamestjw) [inline install.php:18]: I ultimately realised that this offered the most flexibility

File: js/discussion.js [hunk 2]
```diff
@@ -4,3 +4,40 @@ $(".replyMessage")
...
+        e.preventDefault();
+        const data = Object.fromEntries(new FormData(e.target).entries());
+
+        // TODO: How to handle failure?
+        var request = new XMLHttpRequest();
+        request.open("POST", "poll.php");
+        request.send(new FormData(e.target));
+        location.reload();
+    });
+}
```
Comment (jamestjw) [inline js/discussion.js:43]: the jquery version of this would 've been a lot more pretty and user friendly  😭

File: public/uploadStudentList.php [hunk 3]
```diff
@@ -1,53 +1,45 @@
...
-                    $create_success = true;
-                }  catch(PDOException $error) {
-                     echo "<br>" . $error->getMessage();
-                }
-            
-    
+        while (($studentData = fgetcsv($students, 1000, ",")) !== false) {
+            $user = new User();
+            $user->student_id = $studentData[0];
+            $user->first_name = $studentData[1];
```
Comment (jamestjw) [inline public/uploadStudentList.php:15]: I have been running a linter locally, and I recommend you guys to do the same :) it helps a lot with formatting (php code is super messy as it is so I find this very handy)

https://github.com/FriendsOfPHP/PHP-CS-Fixer heres what I use

**Review by chris7neves — APPROVED** [review]
Looks very good, beefy commit

**Pull request merged**